etna/packages/etna-js/upload/workers/uploader.jsx

```diff
@@ -102,6 +102,11 @@ export function Upload({
     current_byte_position = 0,
     next_blob_hash = ZERO_HASH,
     file_size = file.size,
+    metis_uid,
```

Comment:
Need to add these values to the `Upload` class, so that the right `file_attribute` and `image_attribute` entries can be found in Timur, since there can be multiple uploads on a single page running at any given time.